superset-frontend/.eslintrc.js

```diff
@@ -114,6 +114,12 @@ module.exports = {
         },
       },
     },
+    {
+      files: ['*.stories.jsx','*.stories.tsx'],
+      rules: {
+        'import/no-extraneous-dependencies': 0,
```

Comment:
could you add a comment on why we disable this rule for stories?

Reply:
Good idea... and actually, I can make the rule disabling a little more specific, only disabling it for devDependencies.